stdlib/public/stubs/Stubs.cpp

```diff
@@ -33,6 +33,20 @@
 #include <sstream>
 #include <cmath>
 #define fmodl(lhs, rhs) std::fmod(lhs, rhs)
+#elif defined(__ANDROID__)
+#include <locale.h>
+static double swift_strtod_l(const char *nptr, char **endptr, locale_t loc) {
```

Comment:
Added a comment. As far as I know from others' comments and from [the source code](https://android.googlesource.com/platform/bionic/+/ndk-r11c/libc/bionic/locale.cpp#40), I think it is true that Android does not support locales other than C. Of course, the Android SDK allows applications to be localized, but this support does not exist at the libc (Bionic) level.